src/spi.rs

```diff
@@ -13,15 +13,25 @@
 //!
 //! use embedded_hal::blocking::spi::{Transfer, Write};
 //! use embedded_hal_mock::spi::{Mock as SpiMock, Transaction as SpiTransaction};
-//!
+//! use embedded_hal::spi::FullDuplex;
+//! 
 //! // Configure expectations
 //! let expectations = [
+//!     SpiTransaction::send(0x09),
```

Comment:
Thanks for adding the example, however I would add this `FullDuplex`example in addition (separately) to the original example as I would say that they are separate concepts and they would be easier to understand separately.

Reply:
They are not separate concepts, it it possible to interchange freely between using Fullduplex send/read-pairs and transfer. I wanted to highlight this from the example, feel free to edit the example to your liking there is code below in the tests section you could cut and paste from.